utils/sysfs/sysfs.go

```diff
@@ -335,25 +352,205 @@ func (fs *realSysFs) GetSystemUUID() (string, error) {
 	}
 }
 
-func (fs *realSysFs) IsCPUOnline(dir string) bool {
-	cpuPath := fmt.Sprintf("%s/online", dir)
-	content, err := ioutil.ReadFile(cpuPath)
+func (fs *realSysFs) IsCPUOnline(cpuPath string) bool {
+	return isCPUOnline(cpuPath)
+}
+
+// isCPUOnline verifies if a CPU in online. Takes path to /sys/devices/system/cpu/cpu*/ directory as a parameter.
```

Comment:
aren't we actually checking `/sys/bus/cpu/devices/` which is `cpuBusPath`? 

what is difference between `/sys/bus/cpu/devices/` and `/sys/devices/system/cpu/cpu*/` ...?

Reply:
We are, you are right. `/sys/bus/cpu/devices/` consists of symlinks to `/sys/devices/system/cpu/cpu*/`, I think.